src/herbivoresAndCarnivores.js

```diff
@@ -1,19 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor (name, health = 100, lives = true) {
```

Comment:
```suggestion
  constructor (name, health = 100) {
```
Why do you need `lives` boolean? It's actually redundant